transport/peers/peer.go

```diff
@@ -0,0 +1,82 @@
+package peers
```

Comment:
IMO you should avoid plurals in the package name unless it's providing utility
functions for a specific type (like `strings`, `bytes`).